ansible/roles/test/tasks/continuous_link_flap.yml

```diff
@@ -214,3 +214,6 @@
     - name: perform config reload
       shell: config reload -y
       become: true
+
+    - name: wait 30 secs for all ports up
```

Comment:
@msosyak consider using "smart" timeout.

Reply:
what is smart timeout?

Reply:
@lguohan having some condition check along with the timeout value: if true - skip the rest of the timeout.